### PR TITLE
Skip autopublish if disabled in MTC config

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [na] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

This PR introduces a new MTC config flag `disableAutoPublish`.
If set to `true`, which would be the case almost all of the time on development machines, no auto-publishing of feeds to the external publisher would occur.

### CODE REVIEW

**Review by daniel-heppner-ibigroup — APPROVED** [review]

**Review by br648 — APPROVED** [review]
Noticed that this required a second review. LGTM.

**Pull request merged**
